### ISSUE TITLE
Improve timing and content of notifications

### ISSUE THREAD

**Issue by phpai** [openkfw/TruBudget#119]
In GitLab by @lennart.ploen on Mar 12, 2019, 14:58

**Objective**: Contact assignee at right moment with clear instructions

**Concept**: 
- Timing : A notification should be sent out to the user at the moment when 2 conditions are satisfied: 1) the workflow item has been assigned to the user and 2) the preceding workflow item has been closed. This way we make sure that at the moment of notification, the user is able to close the workflow item straight away on TB (not the case if condition 2 is not satisfied; which is currently the case).
- Content : "Workflow item ______ (name of workflow item) has been assigned to you on TruBudget. Please validate the document ______________(name of document) attached to the workflow item (if applicable) and close the workflow item once the described task has been accomplished. Link to workflow item : _______________ (please make sure to be logged into TruBudget so that the link opens correctly).
- Note that notifications should be sent out on TB and via email (see issue 335 and 346)

**Comment by phpai** [comment]
In GitLab by @lennart.ploen on Mar 12, 2019, 16:39

changed the description

**Comment by gonzochic** [comment]
Relates to #65

**Comment by mathiashoeld** [comment]
postponed as issue is dependent on #77
